src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.strategy.AddOperationHandler;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static Parser parser;
+    private static FruitService fruitService;
+    private static FileReader reader;
```

Comment:
instead of depending on so many services, let's fill the storage with our hands in this test
and then check the report

Reply:
edited ✔️